pkgs/tools/admin/trivy/default.nix

```
@@ -51,6 +67,7 @@ buildGoModule rec {
       vulnerabilities of OS packages (Alpine, RHEL, CentOS, etc.) and
       application dependencies (Bundler, Composer, npm, yarn, etc.).
     '';
+    mainProgram = pname;
```

```suggestion
    mainProgram = "trivy";
```
The purpose of this fields is to no inherit pname